docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+version: '3.5'
+services:
+  xblock_sdk:
+    image: xblock-sdk
+
+    # Select the appropriate "build:" block:
```

Comment:
Can you please remove all the unneeded commented out code?

Reply:
Done